friends/src/App.js

```diff
@@ -49,24 +49,24 @@ class App extends Component {
       name: this.state.friendName,
       age: this.state.friendAge,
       email: this.state.friendEmail
-    }
+    };
     axios.post('http://localhost:5000/friends', newfriend)
     .then(result => {
       this.setState({
       friendName: '',
-      friendAge: undefined,
```

Comment:
If you want to leave something undefined. `null`. However, `' '`, is more suited for this use case as far as I know. Well done!

Reply:
Correction. Further research seems to conclude that `null` would be a more proper value for the initial state definition of `friendAge`

Reply:
I tried using null, but it wouldn't let me. I'll try again!